legal/refund-policy.md

```diff
@@ -0,0 +1,44 @@
+---
+title: Refund Policy
+layout: en
+no_toc: true
+---
+
+# Upgrades, Downgrades, Plan Changes
```

Comment:
We take the `title: Refund Policy` as H1, so all of these headings need to be reduced one level `#` -> `##` and `##` -> `###`.